src/pages/chatting/chatting.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Chatting = ({ users, setUsers }) => {
```

Comment:
```suggestion
const Chatting = () => {
```

쓰이지 않을 `props`는 안넣는것이 아무래도 가독성에 좋겠죠?